Expand `Overmapping` article with examples and more contemporary explanations (#12561)
ppy/osu-wiki: wiki/Beatmapping/Overmapping/en.md

# Overmapping

> In beatmapping, **overmapping** is a mapping technique where [hit objects](/wiki/Gameplay/Hit_object) are deliberately placed at a much higher contrast compared to what the song (or the rest of the map) conceivably calls for.

Author: oh yes these are the words i've been looking for, tyty

> This is mainly done by mapping to inaudible or non-existant sounds (i.e. adding notes on blue ticks when the song only uses red ticks). Less frequently, overmapping can also be achieved by adding exorbitantly large [jumps](/wiki/Beatmap/Pattern/osu!/Jump) or sliders with very high [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity).

Reviewer: ```suggestion
This is mainly done by mapping to inaudible or non-existant sounds (e.g. adding notes on blue ticks when the song has sounds only on red ticks). Less frequently, overmapping can also be achieved by adding exorbitantly large [jumps](/wiki/Beatmap/Pattern/osu!/Jump) or sliders with very high [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity).
```

## Examples

![Example of overmapping by excessively high object density](img/overmap-stream.gif "Example of overmapping by excessively high object density")

![Example of overmapping by excessively high object spacing](img/overmap-jump.gif "Example of overmapping by excessively high object spacing")

![Example of overmapping by excessively high slider velocity](img/overmap-slider.gif "Example of overmapping by excessively high slider velocity")

> Please note that while overmapping is generally discouraged, this technique remains perfectly acceptable to use when applied under an appropriate context.

Reviewer: i wonder if this can be worded in such a way that it doesn't sound like a vague opinion from the author, perhaps by adding concrete examples

```suggestion
While overmapping is generally discouraged, there are contexts where the technique is acceptable.
```

Author: yeah

> While overmapping by itself is generally discouraged, there are contexts where the technique is acceptable such as when the added rhythm [creates an emphasis that benefits the map and the players alike](https://osu.ppy.sh/community/forums/posts/7791118).

Reviewer: so
- additive rhythm
- benefits map (quality, not pp gain)

```suggestion
While overmapping by itself is generally discouraged, there are contexts where the technique is acceptable, such as when the added rhythm [fits the song and creates emphasis that benefits the map's quality](https://osu.ppy.sh/community/forums/posts/7791118).
```
